tracer/missing-nullability-files.csv

```diff
@@ -200,6 +200,7 @@ src/Datadog.Trace/Tagging/AspNetCoreMvcTags.cs
 src/Datadog.Trace/Tagging/AspNetCoreTags.cs
 src/Datadog.Trace/Tagging/AspNetTags.cs
 src/Datadog.Trace/Tagging/AwsDynamoDbTags.cs
+src/Datadog.Trace/Tagging/AwsEventBridgeTags.cs
```

Comment:
This is a list of files that still need `#nullable enabled`. We're trying to to make this file smaller, not larger, so we kinda require nullability annotations an all new files going forward.

Can you add nullability annotations to `AwsEventBridgeTags.cs` so it doesn't get added to this list?

Reply:
Ah I was confused by that file, thanks for the clarification.